plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -63,30 +63,24 @@ class LocalParams(BaseParams):
     pass
 
 
-@dataclass
-class SlurmConf(ObjectConf):
-    cls: str = "hydra_plugins.hydra_submitit_launcher.submitit_launcher.SlurmSubmititLauncher"
-    params: SlurmParams = SlurmParams()
-
-
-@dataclass
-class LocalConf(ObjectConf):
-    cls: str = "hydra_plugins.hydra_submitit_launcher.submitit_launcher.LocalSubmititLauncher"
-    params: LocalParams = LocalParams()
-
-
 # finally, register two different choices:
 ConfigStore.instance().store(
     group="hydra/launcher",
     name="submitit_local",
-    node=LocalConf,
+    node=ObjectConf(
+        cls="hydra_plugins.hydra_submitit_launcher.submitit_launcher.LocalLauncher",
+        params=LocalParams(),
+    ),
```

Comment:
Not sure why it failed for me then :s maybe just because it was getting late :s

Reply:
I am curious as well, if you find out what it was let me know :)